tests/providers/aws/services/awslambda/awslambda_function_no_secrets_in_variables/awslambda_function_no_secrets_in_variables_test.py

```diff
@@ -115,6 +115,49 @@ def test_function_secrets_in_variables(self):
             )
             assert result[0].resource_tags == []
 
+    def test_function_secrets_in_variables_telegram_token(self):
+        lambda_client = mock.MagicMock
+        function_name = "test-lambda"
+        function_runtime = "nodejs4.3"
+        function_arn = f"arn:aws:lambda:{AWS_REGION_US_EAST_1}:{AWS_ACCOUNT_NUMBER}:function/{function_name}"
+
+        lambda_client.functions = {
+            "function_name": Function(
+                name=function_name,
+                security_groups=[],
+                arn=function_arn,
+                region=AWS_REGION_US_EAST_1,
+                runtime=function_runtime,
+                environment={"TELEGRAM_BOT_TOKEN": "telegram-token"},
```

Comment:
You should add a valid Telegram Bot Token because detect-secrets flags it as a secret for the environment value not by their name.